.mailmap

```diff
@@ -1396,6 +1396,7 @@ dranknight09 <cbhaavan@gmail.com>
 dustyrockpyle <dustyrockpyle@gmail.com>
 elvis-sik <e.sikora@grad.ufsc.br>
 erdOne <36414270+erdOne@users.noreply.github.com>
+extraymond <extraymond@gmail.com>
```

Comment:
@extraymond, how do you want your name to be recorded in the AUTHORS file?

Reply:
Thanks for mentioning it, it's my pleasure to be included.

Reply:
> Thanks for mentioning it, it's my pleasure to be included.

Oscar means, whether you want your real name shown, or just github user name. And if real name, what is it.

Reply:
Oh! Handle should be fine.